lib/vanagon/common/pathname.rb

```diff
@@ -28,7 +30,7 @@ class Pathname
       #   and exposed through the {#configfile?} method.
       # @return [Vanagon::Common::Pathname] Returns a new Pathname instance.
       def initialize(path, mode: nil, owner: nil, group: nil, config: false)
-        @path = File.expand_path(path)
+        @path = ::Pathname.new(path).cleanpath
```

Comment:
I think we'll have to revisit this some day or write good doc on it, just because of the joys of interpolated strings vs. literal ones:

```ruby
>> path = Pathname.new('C:\some thing\some where')
=> #<Pathname:C:\some thing\some where>
>> path = Pathname.new("C:\some thing\some where")
=> #<Pathname:C: ome thing ome where>
```